### DISCUSSION TITLE
Update to latest TailPress version without impacting customized files

### DISCUSSION THREAD

**Discussion by ktoombs-sm2dev** [tailpress/tailpress#95] [General]
Hi Jeffrey, Is there a way to update the files that you have released updates for recently without overwriting the custom work I have in place? I have used your boilerplate on a few projects and want to make sure the most recent code is in place. I'm curious to know if there is an automated way of doing this, or if I will need to manually implement the recent changes you have made.

**Comment by jeffreyvr** [comment]
Hi, there is no automated way at the moment. For myself, I just manually make the changes to themes I made with older TailPress versions. And I only changes the things I really need.

Maybe I can make a simple upgrade guide for future releases.

**Reply by ktoombs-sm2dev** [reply]
Okay. Thank you for confirming.

**Comment by ethan-tqa** [comment]
Just my 2 cents: If the version jump is small enough, I can just paste the new tailpress files over and let git tells me what have changed. But if I see a lot of changes, or if I have not updated in a long time, I would start with a fresh copy of tailpress and manually migrate my customizations over.

**Reply by ktoombs-sm2dev** [reply]
I appreciate hearing your thoughts, @ethan-tqa. I wasn't sure if there was any other way that Jeffrey had already implemented, but checking version changes and pasting them over is the way I figured I'd go about it.
